data/data/bootstrap/files/usr/local/bin/installer-gather.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+ARTIFACTS="${1:-/tmp/artifacts}"
+mkdir -p "${ARTIFACTS}"
+
+echo "Gathering bootstrap journals ..."
+mkdir -p "${ARTIFACTS}/bootstrap/journals"
+for service in bootkube openshift kubelet crio
+do
+    journalctl --boot --no-pager --output=short --unit="${service}" > "${ARTIFACTS}/bootstrap/journals/${service}.log"
+done
+
+echo "Gathering bootstrap containers ..."
+mkdir -p "${ARTIFACTS}/bootstrap/containers"
+for container in $(crictl ps --all --quiet)
+do
+    container_name=$(crictl ps -a --id ${container} -v | grep -oP "Name: \K(.*)")
+    crictl logs "${container}" >& "${ARTIFACTS}/bootstrap/containers/${container_name}.log"
```

Comment:
We probably want to at least attempt to include logs from previous attempts at this container, via [`--last`][1] with some kind of suffix here.

[1]: https://github.com/kubernetes-sigs/cri-tools/blob/v1.14.0/cmd/crictl/container.go#L307-L310